n6001/hardware/ofs_n6001/build/opencl_afu.json

```diff
@@ -2,8 +2,8 @@
    "version": 1,
    "afu-image": {
       "power": 0,
-      "clock-frequency-high": "auto",
```

Comment:
These constraints are now used by the FIM's version of adjust-plls. This tells the script(s) to find the highest clean Fmax up to 600MHz/1200MHz.